linera-core/src/local_node.rs

```diff
@@ -142,8 +156,24 @@ where
 
 impl<S> LocalNodeClient<S> {
     pub fn new(state: WorkerState<S>, notifier: Arc<Notifier<Notification>>) -> Self {
+        use rand::SeedableRng as _;
+
+        #[cfg(target_arch = "wasm32")]
+        let rng = {
+            // The following seed is chosen to have equal numbers of 1s and 0s, as advised by
+            // https://docs.rs/rand/latest/rand/rngs/struct.SmallRng.html
+            // Specifically, it's "01" × 32 in binary
+            const RNG_SEED: u64 = 6148914691236517205;
+
+            rand::rngs::SmallRng::seed_from_u64(RNG_SEED)
```

Comment:
`linera-core` doesn't need to compile for non-web Wasm, so we should be able to use an external source of randomness

Reply:
I have done this — although it makes our build much more complicated again (I've had to re-introduce the tree of `web` features).  I'm not sure which is better :)  But we can think about it again at a later date.